docs/useCanAccess.md

```diff
@@ -5,15 +5,14 @@ title: "useCanAccess"
 
 # `useCanAccess`
 
-This hook, part of [the ra-rbac module](https://react-admin-ee.marmelab.com/documentation/ra-rbac)<img class="icon" src="./img/premium.svg" />, calls the `authProvider.getPermissions()` to get the role definitions, then checks whether the requested action and resource are allowed for the current user. 
+This hook calls the authProvider.canAccess() method using react-query for a provided resource and action (and optionally a record).
```

Comment:
```suggestion
This hook calls the `authProvider.canAccess()` method on mount for a provided resource and action (and optionally a record). It returns an object containing a `canAccess` boolean set to `true` is the current user can access the resource and action. 
```

The fact that it uses react-query is an implementation detail IMHO

Reply:
That was so users expect react-query properties in the result but ok